main.py

```diff
@@ -0,0 +1,51 @@
+originArray = [[1,2,3],[4,5,6],[7,8,9]]
+finalArray = []
+
+# append to final array leftmost column of origin array
+def goUp(a):
+    for i in range(0, len(a)):
+        finalArray.append(a[len(a)-1-i].pop(0))
```

Comment:
Python supports negative indexing, so a[-1] would return you the last item. Can you think of better way to write this line?